content/communityExtensions/figmaToBabylonJS.md

```diff
@@ -0,0 +1,69 @@
+![Screenshot showing the figma right click menu on the Plugins option](/img/extensions/FigmaToBabylonJS-Step2.png)
+
+It's also good practice to load the UI in gui.babylonjs.com or in your project but loading in the inspector so you can edit the UI after the fact. Just to make sure that the exporting has went the way you expected and there weren't any strange inconsistencies with the design [see inconsistencies and polyfills](/communityExtensions/figmaToBabylonJS#inconstancies--polyfills)
+
+## How it Works
+
+The plugin recursively loops through the selected Figma frame and pulls out all the "nodes" in the design. It will take each node and find a correct BabylonJS GUI object to map to it.
+All the new BabylonJS GUI mapped objects are then added to a empty root container object as children. When the plugin is finished adding all these children, it simply outputs it as a JSON string and writes it to a file and makes it available to download or “stream” from Figma directly.
+
+## Inconstancies & Polyfills
+
+Figma has features that beyond the scope of BabylonJS GUI system. Below is a list of some of the areas which the plug-in has to make a determination on how to interpret things that Figma is asking for that BabylonJS GUI cannot replicate 1:1.
+
+### Handling Images
+
+Images can’t be represented as binary or BASE64 inside the json file so currently the plugin will just set the image src too `source=“./imageControlDefault.jpg”`. After exporting the json its recommended you export your images separately and correct the source addresses either in the JSON file or in code.
+
+### Misc Inconstancies
+
+- Borders
+  - All borders are treated as "Outer" when porting to BabylonJS. 
+- Colors not 1:1
+  - By default Figma uses an "unmanaged" color space, to make Figma closer represent what will actually be outputted when viewed on the web browser, go to Figma preferences > ColorSpace and set the ColorSpace to sRGB.
+- Polygons
+  - Polygons are not supported by BabylonJS GUI, the plug-in will simply skip over polygons when encountered. If you need to use complex shapes and polygons like this it's recommended that you export it as its own PNG and imported into the BabylonJS project as an image.
+- Lines
+  - Lines are supported by BabylonJS but Figma represents them as an array of vertexes. Right now the plugin will skip over lines but this feature will be added in future.
+- Gradient Fills
+  - This feature is being worked on, currently the plug-in will simply skip over any gradient used, this will be supported in future.
+- Blending/Multi Fills
+  - Figma allows you to layer fills one on top of the other and use special blending options. Right now the plugin will only use the top most fill. 
+  - Do not use fills that have its visibility turned off. There have been occasions where Figma returns false positives.
```

Comment:
There is an inconsistency using the plural "fills" and the singular "its." The correction here would be either:

"Do not use a fill that has its visibility turned off."

or

"Do not use fills that have their visibility turned off."

Reply:
Went with the second option as you can have many fills.